src/App.test.jsx

```diff
@@ -64,7 +72,9 @@ it('should count and reset all the things', () => {
   const incrementButton = screen.getByRole('button', { name: 'increment' })
   userEvent.click(incrementButton)
   expect(counter).toHaveTextContent('1')
-  expect(counter).toHaveStyle(`color: ${green}`)
+  setTimeout(() => {
```

Comment:
These timeouts should not be needed with the color being updating together with the count in the reducer